cmd/kubeadm/app/cmd/init.go

```diff
@@ -584,14 +584,7 @@ func waitForAPIAndKubelet(waiter apiclient.Waiter) error {
 
 	go func(errC chan error, waiter apiclient.Waiter) {
 		// This goroutine can only make kubeadm init fail. If this check succeeds, it won't do anything special
-		if err := waiter.WaitForHealthyKubelet(40*time.Second, "http://localhost:10255/healthz"); err != nil {
-			errC <- err
-		}
-	}(errorChan, waiter)
-
-	go func(errC chan error, waiter apiclient.Waiter) {
-		// This goroutine can only make kubeadm init fail. If this check succeeds, it won't do anything special
-		if err := waiter.WaitForHealthyKubelet(60*time.Second, "http://localhost:10255/healthz/syncloop"); err != nil {
```

Comment:
I'm not sure about the history of why we were testing both /healthz and /healthz/syncloop on the read only port? Do we introduce any type of regression by only testing against /healthz using the healthz endpoint?

Reply:
For no specific reason really. I added it :wink:. I just found two endpoints, so I thought two were better than one. Anyway, we're planning to depend less on this when we are adding the prepulling of the images via CRI instead (cc @chuckha). In the end this is just providing better UX if the kubelet can't start cleanly. `/healthz` should catch that